src/multiple_choice/config.json

```
@@ -0,0 +1,6 @@
+{
+    "answerColoring": "DEFAULT_COLORING",
+    "colorQuestionTable": "false",
+    "colorAnswerTable": "true",
+    "hideAnswerTable": "false"
```

In my opinion these should be bool, not string.
Requires some changes in `getOptionsJavaScriptFromConfig` too.
```suggestion
    "colorQuestionTable": False,
    "colorAnswerTable": True,
    "hideAnswerTable": False
```